README.md

```diff
@@ -1 +1,3 @@
-Introduction to Git and GitHub
\ No newline at end of file
+Introduction to Git and GitHub.
+
+This is my first lab activity in CTINPRGL: Integrative Programming and Technologies
```

Comment:
Please include an additional sentence that explains the purpose or goal of this lab activity.